### PR TITLE
[feature]: Added Validate() function to check if the linux kernel version is lower than 5.15

### PR DESCRIPTION
**Pull request by Akash-Singh04**
## Related Issue
  - Added Validate() function to check if the linux kernel version is lower than 5.15 for eBPF support. 

Closes: #1811 

#### Describe the changes you've made
Added Validate() function to check if the linux kernel version is lower than 5.15 for eBPF support. Validate() then calls ValidateFlags() for record and test command.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added
NIL
Please describe the tests(if any). Provide instructions how its affecting the coverage.
NIL
#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.
NIL
## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by Akash-Singh04** [comment]
@slayerjain @gouravkrosx Kindly review

**Comment by gouravkrosx** [comment]
@Akash-Singh04, You can refer this: https://pkg.go.dev/github.com/moby/moby/pkg/parsers/kernel#section-documentation

**Comment by Akash-Singh04** [comment]
@gouravkrosx your requested changes have been integrated

File: cli/provider/cmd.go [hunk 1]
```diff
@@ -240,6 +241,18 @@ func (c *CmdConfigurator) AddFlags(cmd *cobra.Command) error {
 	return nil
 }
 
+func (c *CmdConfigurator) Validate(ctx context.Context, cmd *cobra.Command) error {
+	//check if the version of the kernel is above 5.15 for eBPF support
+	isValidVersion := kernel.CheckKernelVersion(5, 15, 0)
```
Comment (gouravkrosx) [inline cli/provider/cmd.go:246]: just call it `isValid`

**Review by gouravkrosx — APPROVED** [review]
LGTM

**Pull request merged**